index.html

```diff
@@ -4,8 +4,9 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="style.css">
-  <script src="script.js" defer></script>
+  <link rel="stylesheet" href="index.css">
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/luxon/2.0.2/luxon.min.js"></script>
+  <script type="module" src="index.js"></script>
```

Comment:
- [ ] [OPTIONAL] Kindly move JavaScript links to the bottom ↴ Because it's generally recommended to place JavaScript links at the end of the body section to ensure the page's HTML content loads before JavaScript execution 🎯  This can improve the page's loading speed ⌛